examples/mnasnet/mnasnet.py

```diff
@@ -0,0 +1,324 @@
+"""
+Training Classifier, Export & Benchmark using Vortex
```

Comment:
would you mind checking your rst formatting? in rst inline code is using a double tick (``like this``), and also indentation matters, especially for bullet and numbering.

Reply:
backtick updated, list changed to table